### ISSUE TITLE
Flathub distro

### ISSUE THREAD

**Issue by 86ul** [bardsoftware/ganttproject#2188]
Hello dev of GanttProject,

Is it possible to turn this project into a Flathub aswell?

Thanks

**Comment by dbarashev** [comment]
No, unless there is someone who is willing to invest their time into it.
